drivers/wifi/nrf_wifi/src/wpa_supp_if.c

```diff
@@ -1595,46 +1599,57 @@ void nrf_wifi_wpa_supp_event_get_wiphy(void *if_priv,
 
 	vif_ctx_zep = if_priv;
 	rpu_ctx_zep = vif_ctx_zep->rpu_ctx_zep;
+	vif_ctx_cnt = nrf_wifi_fmac_get_num_vifs(rpu_ctx_zep->rpu_ctx);
 
 	memset(&band, 0, sizeof(band));
 
-	for (int i = 0; i < NRF_WIFI_EVENT_GET_WIPHY_NUM_BANDS; i++) {
-		if (nrf_wifi_parse_sband(&wiphy_info->sband[i], &band) != WLAN_STATUS_SUCCESS) {
+	for (int idx = 0; idx < vif_ctx_cnt; ++idx) {
+		vif_ctx_zep = nrf_wifi_get_vif_ctx_by_idx(idx);
+		if (vif_ctx_zep == NULL) {
 			continue;
 		}
-		if (vif_ctx_zep->supp_drv_if_ctx && vif_ctx_zep->supp_callbk_fns.get_wiphy_res) {
-			vif_ctx_zep->supp_callbk_fns.get_wiphy_res(vif_ctx_zep->supp_drv_if_ctx,
-									&band);
+
+		for (int i = 0; i < NRF_WIFI_EVENT_GET_WIPHY_NUM_BANDS; i++) {
+			if (nrf_wifi_parse_sband(&wiphy_info->sband[i], &band) !=
+				WLAN_STATUS_SUCCESS) {
+				continue;
+			}
+			if (vif_ctx_zep->supp_drv_if_ctx &&
+				vif_ctx_zep->supp_callbk_fns.get_wiphy_res) {
+				vif_ctx_zep->supp_callbk_fns.get_wiphy_res(
+					vif_ctx_zep->supp_drv_if_ctx, &band);
+			}
 		}
-	}
 
-	if ((wiphy_info->params_valid & NRF_WIFI_GET_WIPHY_VALID_EXTENDED_CAPABILITIES) &&
-	    rpu_ctx_zep->extended_capa == NULL) {
-		/* To avoid overflowing the 100 column limit */
-		unsigned char ec_len = wiphy_info->extended_capabilities_len;
+		if ((wiphy_info->params_valid & NRF_WIFI_GET_WIPHY_VALID_EXTENDED_CAPABILITIES) &&
+			rpu_ctx_zep->extended_capa == NULL) {
```

Comment:
This shows an issue with indentation. The indentation should be like this here
```
		if ((wiphy_info->params_valid & NRF_WIFI_GET_WIPHY_VALID_EXTENDED_CAPABILITIES) &&
		    rpu_ctx_zep->extended_capa == NULL) {
```
in order to follow the coding style in zephyr.